webrtc-c/canary/src/Include.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#define DEFAULT_CLOUDWATCH_NAMESPACE "KinesisVideoSDKCanary"
+#define DEFAULT_FPS_VALUE            25
+// TODO: This value shouldn't matter. But, since we don't allow NULL value, we have to set to a value
+#define DEFAULT_VIEWER_PEER_ID "ConsumerViewer"
+
+#define MAX_LOG_STREAM_NAME        512
+#define MAX_CLOUDWATCH_LOG_COUNT   128
+#define MAX_CONCURRENT_CONNECTIONS 10
+#define MAX_TURN_SERVERS           1
+
+#define NUMBER_OF_H264_FRAME_FILES  1500
+#define NUMBER_OF_OPUS_FRAME_FILES  618
+#define SAMPLE_VIDEO_FRAME_DURATION (HUNDREDS_OF_NANOS_IN_A_SECOND / DEFAULT_FPS_VALUE)
+#define SAMPLE_AUDIO_FRAME_DURATION (20 * HUNDREDS_OF_NANOS_IN_A_MILLISECOND)
+
+#define ASYNC_ICE_CONFIG_INFO_WAIT_TIMEOUT (3 * HUNDREDS_OF_NANOS_IN_A_SECOND)
+#define ICE_CONFIG_INFO_POLL_PERIOD        (20 * HUNDREDS_OF_NANOS_IN_A_MILLISECOND)
+
+#define CANARY_CHANNEL_NAME_ENV_VAR    "CANARY_CHANNEL_NAME"
```

Comment:
One thing I have in mind: Should we create a shell script with exports of all this so that all the customer needs to do is modify values in the script and run it? 

I feel it will be a lot more cleaner and easier to keep track of while running the app.

Reply:
That was somewhat my plan. My vision is that we'll have a script that will enumerate the permutation that we want to test. By having this script, it'll also expose the possible environment variables to the customers.